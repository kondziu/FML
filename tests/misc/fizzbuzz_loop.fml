// Expected output:
// > 1
// > 2
// > 3 fizz
// > 4
// > 5 buzz
// > 6 fizz
// > 7
// > 8
// > 9 fizz
// > 10 buzz
// > 11
// > 12 fizz
// > 13
// > 14
// > 15 fizz buzz
// > 16 not considered

let i = 1;
while i <= 15 do
begin
  print("~", i);
  if i % 3 == 0 then print(" fizz");
  if i % 5 == 0 then print(" buzz");
  print("\n");
  i <- i + 1;
end;

print("~ not considered\n", i);