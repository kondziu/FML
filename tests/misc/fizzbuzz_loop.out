1
2
3 fizz
4
5 buzz
6 fizz
7
8
9 fizz
10 buzz
11
12 fizz
13
14
15 fizz buzz
16 not considered
